modules/programs/rbw.nix

```diff
@@ -87,7 +81,7 @@ in {
         {
           email = "name@example.com";
           lock_timeout = 300;
-          pinentry = "gnome3";
+          pinentry = pkgs.pinentry-gnome;
```

Comment:
Upstream seems to be only `pinentry-gnome3` ?

Reply:
Wait no, my branch might be too old.

Reply:
So this might not work on `unstable`? `unstable` has been renamed to `pinentry-gnome3` `23.11` is still `pinentry-gnome`

Reply:
This is just an example...